<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QBinCompareDialog</class>
 <widget class="QDialog" name="QBinCompareDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>file1:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="file1Path"/>
       </item>
       <item>
        <widget class="QPushButton" name="browse1">
         <property name="text">
          <string>browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>file2:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="file2Path"/>
       </item>
       <item>
        <widget class="QPushButton" name="browse2">
         <property name="text">
          <string>browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextBrowser" name="output"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="compare">
         <property name="text">
          <string>compare</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="quit">
         <property name="text">
          <string>quit</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
